### PR TITLE
Add License to the site and repository

### LINKED ISSUE #79: Add a license for website sources and the website text
It's a very nice website layout and solution and others might be interested in reusing this and for this a license would be good. My recommendation is MIT or BSD for the website sources and CC-BY for the content.

### PR DESCRIPTION
**Pull request by Chilipp**
closes #79

As @bast will already be busy with the privacy statement, I am adding the LICENSE stuff here. I suppose we publish the page under the CC BY 4.0 license as discussed during the OC meeting on Wednesday. This PR

* adds a LICENSE file
* adds a copyright disclaimer to the README
* adds a license file to the website (where I also copied the LICENSE information of minimal mistakes, as we incorporate these packages): https://155-267395254-gh.circle-artifacts.com/0/SORSE20/license/index.html
* adds the license to the footer
  image

Did I forget anything @bast and @vsoch?

### CODE REVIEW

File: _pages/license.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: License
+permalink: /license/
+sidebar:
+  nav: legal
+---
+The contents of this website are published under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/) under the
+Copyright © 2020 of the SORSE20 committee.
+
```
Comment (vsoch) [inline _pages/license.md:9]: Any reason to not have the LICENSE as an include (just plop in the includes folder) and then just include it here? if there are changes, we'd want both the rendered and "hard coded" version in the site to be synced.

Comment (vsoch) [reply _pages/license.md:9]: Hmm wait this is listing all the minimal mistakes licenses. Is that preferable to just linking to the minimal mistakes repository, with a summary sentence? We would again need to keep these updated.

**Comment by bast** [comment]
Thanks a lot for taking this! I think this is good to go. One could argue that CC-BY would not really apply to the website source code (but I could also be wrong) and we could say that the website sources are covered by MIT but the website content by CC-BY but in practice the content might be more important.

**Comment by Chilipp** [comment]
Thanks for the feedback @bast. I understand your point but on the other hand, about 90% of the words in this source code are _content_. So it would be strange to give the website a rather strict CC BY license, but then use a relaxed MIT license for the source code. It's difficult to handle though :sweat_smile: I'd just use CC BY for both (or maybe GPL for the source, and CC BY for the website?), but we can also change this later, I think

@vsoch are there any comments from your side?

**Comment by bast** [comment]
I agree Philipp. The CC-BY part is the really important and relevant part.

**Comment by vsoch** [comment]
My 0.02 is that more permissive is better, but in terms of separating software vs. content, I think for the software I would just reference the original author's license and provide a link to it, and then use CC 4 for the content. In summary:

 - content is CC by 4 (and this [seems appropriate](https://en.wikipedia.org/wiki/Creative_Commons_license) - websites is listed as one of the general things it covers)
 - original software we reference a link to the minimal mistakes repository (MIT).

I don't think we've done any new software development here, so I don't think it makes sense to try and put another license for that.

**Comment by Chilipp** [comment]
Hmm, maybe my understanding of a LICENSE in the source code repo is then wrong. If I use an MIT license for the LICENSE file, for instance, wouldn't that mean that the content of a file, such as `_pages/home.md` falls under this MIT license, too? Where it should rather use the CC BY license?

I agree @vsoch that we do not implement much new software development here, which is exactly why I thought that CC BY fits better to the source code repository

**Comment by bast** [comment]
I think it's good as it is (`LICENSE` file is CC-BY) if we add:
> original software we reference a link to the minimal mistakes repository (MIT).

**Comment by Chilipp** [comment]
alright, thank you both for the feedback! this is also a new lesson to me :sweat_smile: 

I included a disclaimer in the LICENSE file that also links to the MIT license of minimal mistakes, and I also included that the `_includes` and `_layouts` folders (this is the part where we do have software development) fall under the MIT license. 0052586fd2afcc5083cc20722c989da332a94a93

This is also stated in the README e50b5891e89a139693224f08d03b911b797b7f20

@vsoch and @bast: if you agree with that, please approve the review and I'll merge this PR

**Comment by Chilipp** [comment]
It might be, that Github has a problem as it may not recognize the license anymore. This is what I get with `licensee detect SORSE20`:

```
License:        NOASSERTION
Matched files:  LICENSE, README.md
LICENSE:
  Content hash:  c25d7e8e69b3284695bd9012bd6ff430e9445b89
  License:       NOASSERTION
  Closest non-matching licenses:
    CC-BY-4.0 similarity:     95.03%
    CC-BY-SA-4.0 similarity:  89.83%
    MPL-2.0 similarity:       36.34%
README.md:
  Content hash:  e857b27db16916db0cad7297a3491b4d8adb8a8c
  Confidence:    90.00%
  Matcher:       Licensee::Matchers::Reference
  License:       CC-BY-4.0
  Closest non-matching licenses:
    WTFPL similarity:  14.81%
    ISC similarity:    13.33%
    0BSD similarity:   13.33%
```

**Comment by bast** [comment]
I would add the notice about MIT into the README but not into the LICENSE. For this reason so that GitHub recognizes it as CC-BY. But I can also do it as follow-up PR.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,403 @@
+Copyright (c) 2020, the SORSE20 committee.
+
+The contents of the SORSE20 website and it's source code repository are licensed
+under the CC BY 4.0 license (see below). The original software for creating this
+website, the Minimal Mistakes Jekyll Theme, is distributed under the MIT
+license, see https://github.com/mmistakes/minimal-mistakes#license, which also
+applies for the _includes and _layouts folder in the source code repository.
```
Comment (Chilipp) [inline LICENSE:7]:
```suggestion
See README.md on how and where to apply this license.
```

> For this reason so that GitHub recognizes it as CC-BY. 

alright, I just link to the README.md in LICENSE. This works fine

> But I can also do it as follow-up PR.

you can also edit the files directly here @bast, e.g. using the `Files changed` tab at the top and click the 
image to make a suggestion (this is what I did here). Then we do not need another PR (see the [github docs](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)).
